### PR TITLE
Create manifest.json

### PR DESCRIPTION
**Pull request by getSono**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Review by DinoMarlir — CHANGES_REQUESTED** [review]
There is one bracket missing. Please update your manifest.

**Comment by getSono** [comment]
> There is one bracket missing. Please update your manifest. 

What bracket?

**Comment by getSono** [comment]
How I do that?
I’m currently on the iPad.

**Comment by getSono** [comment]
And how can I add the } on Web Version? I'm new on GitHub

Gesendet von Outlook für iOS<https://aka.ms/o0ukef>
________________________________
Von: Marlon ***@***.***>
Gesendet: Donnerstag, April 11, 2024 3:41 PM
An: LabyMod/server-media ***@***.***>
Cc: Sono ***@***.***>; Author ***@***.***>
Betreff: Re: [LabyMod/server-media] Create manifest.json (PR #1054)


Just write } at the bottom.

These resources may help you for the future:

  *   https://jsonlint.com/
  *   https://www.w3schools.com/js/js_json_syntax.asp

**Comment by DinoMarlir** [comment]
Yeah. Also you have to provide a icon file.

**Comment by getSono** [comment]
I add the } and add icon.png @DinoMarlir

**Comment by DinoMarlir** [comment]
There are some more errors. Please fix these.

**Comment by DinoMarlir** [comment]
Your icon is invalid too. Please read the documentation and the instructions in the wiki before you push something.

**Comment by DinoMarlir** [comment]
Still invalid. Seems like you don't have much knowledge with Json. I would recommend you to learn the JSON syntax. Then you can push your fix.

**Comment by getSono** [comment]
Yes, I usually never programming with JSON. (lol)
I don't need json in my Java, HTML or Python Projects.

File: minecraft_servers/teamfive/manifest.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "server_name": "teamfive",
+    "nice_name": "TeamFive",
+    "direct_ip": "node2.craftsblock.eu:25583",
+    "supported_languages": [
+       "de"
+    ]
+    }
```
Comment (maaaathis) [inline minecraft_servers/teamfive/manifest.json:8]:
```suggestion
    ],
```

File: minecraft_servers/teamfive/manifest.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "brand": {
+       "primary": "#00A5FF",
+       "background": "#0077B7",
+       "text": "#FFFFFF"
+    }, 
+    "location": {
+       "country": "Germany",
+       "country_code": "DE"
+    }
+  }
```
Comment (maaaathis) [inline minecraft_servers/teamfive/manifest.json:25]:
```suggestion
```

File: minecraft_servers/teamfive/manifest.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "server_name": "teamfive",
+    "nice_name": "TeamFive",
+    "direct_ip": "node2.craftsblock.eu:25583",
+    "supported_languages": [
+       "de"
+    ]
+    }
+    "gamemodes": {
+      "Citybulid": {
+        "name": "Citybulid",
```
Comment (maaaathis) [inline minecraft_servers/teamfive/manifest.json:11]:
```suggestion
      "citybuild": {
        "name": "Citybuild", 
```

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/teamfive/icon.png
Comment (maaaathis) [inline minecraft_servers/teamfive/icon.png:1]: Invalid icon size! Size is 500x500px, must be 256x256px

**Review by maaaathis — CHANGES_REQUESTED** [review]
icon@2x.png is not given, but it is required.

**Comment by DoJapHD** [comment]
The server is just a plain vanilla server?

**Comment by getSono** [comment]
@DoJapHD No, But I currently reinstall the server

**Comment by DoJapHD** [comment]
Reopen PR once server is ready and released

**Pull request closed without merging**